bukuserver/README.md

```diff
@@ -1,3 +1,32 @@
+#### Install server
+You need to have some packages before you install `bukuserver` on your server. So be sure to have `python3`, `python3-pip` , `python3-dev`, `libffi-dev` packages from your distribution.
+##### Installing PIP, virtualenv and dependencies
+```
+$ python3 -m pip install --user --upgrade pip
+$ python3 -m pip install --user virtualenv
+$ python3 -m virtualenv env
+$ source env/bin/activate
+$ pip install appdirs
```

Comment:
imo package should be installed from setup.py same as the next section (or maybe even merge it)

if any package added or removed, the documentation will not be changed